python/GafferUI/GLWidget.py

```diff
@@ -87,6 +87,9 @@ def __init__( self, bufferOptions = set(), **kw ) :
 		if hasattr( format, "setVersion" ) : # setVersion doesn't exist in qt prior to 4.7.
 			format.setVersion( 2, 1 )
 
+		format.setSampleBuffers( True )
```

Comment:
Seems odd to do this unconditionally if it's not going to be used by the NodeEditor and other GLWidgets. Maybe we should either, make it optional via the `bufferOptions` argument, or make sure that we always enable multi-sampling as well?